app/models/application_status.rb

```diff
@@ -0,0 +1,16 @@
+class ApplicationStatus
+  def commit_id
```

Comment:
Could the result of this function use an in memory cache or something to avoid the disk IO each time this end point is hit (it may be used for health checks)?  
https://guides.rubyonrails.org/caching_with_rails.html#activesupport-cache-memorystore
https://guides.rubyonrails.org/caching_with_rails.html#low-level-caching

Reply:
Could store the contents of the file in an ENV or in a Rails.config on load via an initializer, a la https://github.com/zooniverse/Seven-Ten/blob/553db5485d8f6e9f380f264a4832cc6e0d6f68f0/config/initializers/revision.rb

Reply:
Checking out these options 👍

Reply:
@zwolf , @camallen -
I've replicated what was done on the Seven-Ten app linked to by zach (https://github.com/zooniverse/Seven-Ten/blob/553db5485d8f6e9f380f264a4832cc6e0d6f68f0/config/initializers/revision.rb), this solves this issue of having to hit the file server every time.